babel.config.js

```diff
@@ -32,4 +31,14 @@ module.exports = {
       },
     ],
   ].filter(Boolean),
+  overrides: [
+    {
+      exclude: './packages/solid-query/**',
+      presets: ['@babel/react'],
+    },
+    {
+      include: './packages/solid-query/**',
+      presets: ['babel-preset-solid'],
+    },
+  ],
```

Comment:
I'm no babel expert, but I think it should be possible to also have one babel config per package? So we would only keep the shared things in top level and then move the framework specific things further down ?